packages/neon/neon_talk/lib/src/pages/room.dart

```diff
@@ -118,7 +118,8 @@ class _TalkRoomPageState extends State<TalkRoomPage> {
                   );
 
                   if (previousMessage == null ||
-                      (previousMessage.timestamp ~/ _secondsPerDay) != (message.timestamp ~/ _secondsPerDay)) {
+                      (tz.local.translate(previousMessage.timestamp * 1000) ~/ _millisecondsPerDay) !=
+                          (tz.local.translate(message.timestamp * 1000) ~/ _millisecondsPerDay)) {
                     final date = DateTimeUtils.fromSecondsSinceEpoch(tz.UTC, message.timestamp);
```

Comment:
Can't we parse the date directly to the local timezone?

Reply:
Yeah I guess so